### PR TITLE
Emailing gateway

### LINKED ISSUE #19: Transactional mailing gateway
I'm on it.
With Mailjet, in first intention.

### PR DESCRIPTION
**Pull request by tmaziere**
- lib Mailjet
- env vars config for Mailjet API public and private keys
- mask request confirmation function (based on MJ transactional template)

Closes #19

### CODE REVIEW

File: apps/api/src/utils/mailjet.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+          Email: "contact@usinepartagee.fr",
+          Name: "Usine partagée Normandie"
+        },
+        To: [
+          {
+            Email: emailTo,
+            Name: nameTo
+          }
+        ],
+        TemplateID: 1323123,
```
Comment (gaelreyrol) [inline apps/api/src/utils/mailjet.js]: Maybe it's a good idea to put the template ID in an environment variable ?

File: apps/api/src/utils/mailjet.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        Subject: "Demande de masques enregistrée",
+        Variables: {
+          id_request: maskRequestId
+        }
+      }
+    ]
+  });
+  request
+    .then(result => {
+      console.log(result.body);
```
Comment (Kmaschta) [inline apps/api/src/utils/mailjet.js]: Debug console log found :)

File: apps/api/src/utils/mailjet.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    })
+    .catch(err => {
+      console.log(err.statusCode);
```
Comment (Kmaschta) [inline apps/api/src/utils/mailjet.js]: You can use `signale` to log errors

Comment (tmaziere) [reply apps/api/src/utils/mailjet.js]: done!

File: apps/api/src/utils/mailjet.js (hunk 1 above)
Comment (tmaziere) [reply apps/api/src/utils/mailjet.js]: done!

**Review by alexisjanvier — APPROVED** [review]

**Pull request merged**